package.json

```diff
@@ -12,10 +12,13 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "start:silent": "PORT=3001 BROWSER=none react-scripts start",
+    "cypress": "start-server-and-test start:silent http://localhost:3001/ cy:open",
+    "cy:open": "cypress open & yarn start -silent"
```

Comment:
So this feels a bit redundant... Can you please try removing the ` & yarn start -silent` and see if it still works? 

The reason I ask is that you invoke `start:silent` on the line above which is the command you will run to start Cypress.

Reply:
Cool, we did that and it's function